install_config/aggregate_logging.adoc

```diff
@@ -55,21 +62,17 @@ this topic use `logging` as an example project:
 +
 ====
 ----
-$ oadm new-project logging --node-selector label=value
+$ oadm new-project logging --node-selector=""
```

Comment:
We do specifically want an empty selector as otherwise the project could inherit one unexpectedly.